drivers/scheduler/k8s/helmScheduler.go

```diff
@@ -0,0 +1,344 @@
+package k8s
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/gofrs/flock"
+	"github.com/pkg/errors"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/scheduler/spec"
+	"github.com/sirupsen/logrus"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/cli/values"
+	"helm.sh/helm/v3/pkg/downloader"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/repo"
+	"helm.sh/helm/v3/pkg/strvals"
+)
+
+var settings *cli.EnvSettings
+
+// HelmSchedule will install the application with helm
+func (k *K8s) HelmSchedule(instanceID string, options scheduler.ScheduleOptions) ([]*scheduler.Context, *scheduler.HelmRepo, error) {
+	var err error
+
+	// For supporting multiple values to override, will read from a configmap
+	// TODO: Need to change the following hardcoded value for sc too.
+	args := map[string]string{
+		// comma seperated values to set
+		"set": "persistentStorage.storageClassName=central-sc,persistentStorage.enabled=true",
+	}
+
+	// Currently the installation supports for one app in one shot
```

Comment:
Any specific challenge here supporting multiple app charts?

Reply:
Addressed this comment of supporting multiple app charts installation.
Also made helmRepo info as part of the context.